pkg/descriptions.json

```diff
@@ -2132,6 +2132,14 @@
       "System.Web.HttpUtility"
     ]
   },
+  {
+    "Name": "System.Windows.Extensions",
+    "Description": "Provides the functionality to display the X509Certificate2",
```

Comment:
I assume we have grander plans for a package named "System.Windows.Extensions" than "display the X509Certificate2"

Maybe something like "Provides miscellaneous Windows-specific types" at least